ci-operator/step-registry/baremetal/lab/pre/load-balancer/run/baremetal-lab-pre-load-balancer-run-commands.sh

```diff
@@ -78,7 +83,14 @@ for dev in "${devices[@]}"; do
     /sbin/dhclient -v \
     -pf "/var/run/dhclient.$interface.pid" \
     -lf "/var/lib/dhcp/dhclient.$interface.lease" "$interface"
+
+  nsenter -m -u -n -i -p -t "$(podman inspect -f '{{ .State.Pid }}' "haproxy-$CLUSTER_NAME")" \
+    /sbin/dhclient -v -N \
+    -pf "/var/run/dhclient.$interface.pid" \
+    -lf "/var/lib/dhcp/dhclient.$interface.lease" "$interface"
```

Comment:
Do you think we need to differentiate the files for v4 and v6?

Reply:
No

Reply:
so the pid file would be overwritten for sure and we lose the pid of the first instance, don't we? As for the leasefile I'm not sure, maybe worth doing a test.

Reply:
if separation would fix it so let's do it. otherwise, the way suggested before is tested  and working, we can revert it